h2o-logger/src/main/java/water/logging/LoggerFactory.java

```diff
@@ -0,0 +1,56 @@
+package water.logging;
+
+
+
+public class LoggerFactory {
+    private static final String DEFAULT_CLASS_TO_CHECK = "org.slf4j.LoggerFactory";
+    private static final LoggerFactory INSTANCE = new LoggerFactory(DEFAULT_CLASS_TO_CHECK);
+    
+    private final String slf4jClassName;
```

Comment:
we could extend this with waterClassName and look for water.util.Log if slf4j is not found

Reply:
making it this way we get circular dependency between modules with different language levels assigned: genmodel(7)->core(8)->logger(7)->genmodel(7).